src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -483,4 +485,49 @@ public static String convertToEmptyStringIfNull(String str) {
         return (str == null) ? "" : str;
     }
 
+    public static String removeVariables(String statusMessage) {
```

Comment:
What's this for? Public methods should have a header comment unless the method is trivial.